### DISCUSSION TITLE
[Question]: Do I need to create yml file or ENV variables enough for Firebase usage?

### DISCUSSION THREAD

**Discussion by virtuman** [danny-avila/LibreChat#2190] [Q&A]
### What is your question?

I couldn't figure out if I need to set fileStrategy and create a config librechat yml file to enable. 
seems like

### More Details

Would be nice to have CDN_PROVIDER environment settings was enough to enable it. if it doesn't already, and i'm simply not doing something correctly

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
> I couldn't figure out if I need to set fileStrategy and create a config librechat yml file to enable.

You do need it
